src/main/java/com/problem1/myproject/controller/UserRestController.java

```diff
@@ -122,15 +123,15 @@ public UserUpdateDTO deleteUser(@PathVariable long userId){      ///TODO ASK IF
 
     ///requires a json (Coin) and  user id in the path and will append to the portofolio of that user the required coin
     ///return the portofolio of that user
-    @PutMapping("/buy/{userId}")  ///TODO MAYBE ADD QUANTITY LATER
+/*    @PutMapping("/buy/{userId}")  ///TODO MAYBE ADD QUANTITY LATER
     @Transactional
     public List<Coin> buyCoin(@PathVariable long userId, @RequestBody Coin theCoin){
 
 
         return  this.userService.buyCoin(userId,theCoin);
 
 
-    }   ///TODO it doesnt work, i think its harder than i thought
+    }   ///TODO it doesnt work, i think its harder than i thought*/
```

Comment:
@RobertRodeapps this is related to the M-M relation? What exactly doesn't work?

Reply:
pot sa salvez coin ul in portofoliu(atribut a unui User), dar ar trebui salvat si faptul ca Coin -ul e acum detinut de un nou  User si ar trebui sa l adaug pe User ul ce a cumparat coin ul respectiv in lista de holders(atribut a unui Coin)